packages/botonic-react/src/components/message.jsx

```diff
@@ -196,7 +199,43 @@ export const Message = props => {
                 borderLeft: 0,
                 marginLeft: -pointerSize + 1,
               }}
-            />
+              {...otherProps}
+            >
+              <div
```

Comment:
if you had conflicts previously with this PR, maybe you can put it into the description of the PR if this is pending